app/src/main/java/com/example/giftly/FontSizeScreen.java

```diff
@@ -24,12 +24,14 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_font_size_screen);
 
+        //Create variables to connect to xml android:id fields
         Button fontSizeButton = findViewById(R.id.fontSizeButton);
         Button fontSizeButton2 = findViewById(R.id.fontSizeButton2);
         Button fontSizeButton3 = findViewById(R.id.fontSizeButton3);
         Button savingit2 = findViewById(R.id.saveIt2);
         SharedPreferences sharedPreferences;
 
+        //Functionality to set the font size with a value of 16
         fontSizeButton.setOnClickListener(new View.OnClickListener() {
```

Comment:
This may not be easy to expand with new sizes, consider having something iterate through the list or a font size slider for more flexibility.

Reply:
https://medium.com/google-developer-experts/exploring-android-jetpack-preferences-8bcb0b7bdd14

The last parts of this article may help with this implementation

Reply:
I went with the font size slider, and it appears to work perfectly